src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -411,6 +437,13 @@ public CodeFixProvider GetSuppressionFixer(string language, ImmutableArray<Diagn
             return false;
         }
 
+        private bool IsInteractiveCodeFixProvider(CodeFixProvider provider)
+        {
+            // TODO (https://github.com/dotnet/roslyn/issues/4932): Don't restrict CodeFixes in Interactive
+            return provider is AddImport.AbstractAddImportCodeFixProvider ||
```

Comment:
Yes, one possible outcome of this PR is that we don't provide any code actions.  I take it your vote is that this is "worse than nothing"?